pywbem/_listener.py

```diff
@@ -1186,9 +1330,43 @@ def password_prompt():
 
     def stop(self):
         """
-        Stop the WBEM listener threads and callback thread, if they are running.
+        Stop the WBEM listeners, the WBEM listener threads and callback thread,
+        if they are running.
+        """
+        self.stop_servers()
+        self.stop_indication_delivery()
+
+    def stop_indication_delivery(self, force=False):
```

Comment:
When would a user ever stop just the servers or just the delivery?

Reply:
No, they are both stopped.  However, the indication delivery is a separate activity and may also be stopped if there is an exception in the the start of the connections since it was started before the connections are started.

Reply:
ok